conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -76,7 +76,7 @@ dependencies:
 - uvicorn==0.34.*
 - zlib
 - pip:
-  - nvidia_sphinx_theme
+  - nvidia-sphinx-theme
```

Comment:
This package name was incorrect and relied on PyPI's normalization logic to fetch the right thing.